Ciac 10634 prisma cloud extend commands (#35728)
demisto/content: Packs/PrismaCloud/Integrations/PrismaCloudV2/README.md

                "lastUsedTime": "number",
                "status": "string",
                "expiresOn": "number",
                "role": {
                    "id": "string",
                    "name": "string"
                },
                "roleType": "string",
                "username": "string"
            },
            {
                "id": "string",
                "name": "string",
                "createdBy": "string",
                "createdTs": "number",
                "lastUsedTime": "number",
                "status": "string",
                "expiresOn": "number",
                "role": {
                    "id": "string",
                    "name": "string"
                },
                "roleType": "string",
                "username": "string"
            }
        ]
    }
}
```

`prisma-cloud-access-key-disable`

#### Input

| **Argument Name** | **Description**                                              | **Required** |
|-------------------|--------------------------------------------------------------| --- |
| access-key              | Access key ID.                                                | Required | 

> #### Command example

Reviewer: lets add readable output example for those 3 commands

```!prisma-cloud-access-key-disable access-key=id```

`prisma-cloud-access-key-enable`

#### Input

| **Argument Name** | **Description**                                              | **Required** |
|-------------------|--------------------------------------------------------------| --- |
| access-key              | Access key ID.                                                | Required | 

#### Command example
```!prisma-cloud-access-key-enable access-key=id```

`prisma-cloud-access-key-delete`

#### Input

| **Argument Name** | **Description**                                              | **Required** |
|-------------------|--------------------------------------------------------------| --- |
| access-key              | Access key ID.                                                | Required | 

#### Command example
```!prisma-cloud-access-key-delete access-key=id```

## Breaking changes from the previous version of this integration - Prisma Cloud v2
The following sections list the changes in this version.

### Commands
#### The following commands were deprecated in this version because they are not supported by the API anymore:
* ***redlock-list-scans***

> * ***redlock-get-scan-status***

Reviewer: Please revert all changes that are not should be part of you PR

>### Next Page Token:
>token2

`prisma-cloud-access-key-create`

#### Input

| **Argument Name** | **Description** | **Required** |
|-------------------| --- | --- |

> | name              | Access key name | Required | 
> | expires-on             | Timestamp in milliseconds when access key expires. Default:0 | Optional | 
> | service-account-name             | Service account name | Optional |

Reviewer 2: ```suggestion
| name              | Access key name. | Required | 
| expires-on             | Timestamp in milliseconds when access key expires. Default:0. | Optional | 
| service-account-name             | Service account name. | Optional |
```

#### Context Output

| **Path**                  | **Type** | **Description**       |
|---------------------------| --- |-----------------------|

> | PrismaCloud.AccessKeys.id  | String | Access key ID        | 
> | PrismaCloud.AccessKeys.secretKey | String | Access key secret |

Reviewer 2: ```suggestion
| PrismaCloud.AccessKeys.id  | String | Access key ID.        | 
| PrismaCloud.AccessKeys.secretKey | String | Access key secret. |
```

#### Command example
```!prisma-cloud-access-key-create name=MyNewKey```
#### Context Example
```json
{
    "PrismaCloud": {
        "AccessKeys": [
            {
                "Id": "Id",
                "Secret Key": "Secret Key"
            }
        ]
    }
}
```

`prisma-cloud-access-keys-list`

#### Input

| **Argument Name** | **Description** | **Required** |
|-------------------| --- | --- |
| access-key             | Returns the metadata of the access key that has the specified ID. | Optional | 
| limit           | Maximum number of entries to return. | Optional | 

#### Context Output

| **Path**                           | **Type** | **Description**             |
|------------------------------------|----------|-----------------------------|

> | PrismaCloud.AccessKeys.id          | String   | Access key id               | 
> | PrismaCloud.AccessKeys.name        | String   | Name                        |
> | PrismaCloud.AccessKeys.createdBy   | String   | Created By                  | 
> | PrismaCloud.AccessKeys.createdTs   | String   | Time access key was created |
> | PrismaCloud.AccessKeys.expiresOn   | String   | Access key expires on       |
> | PrismaCloud.AccessKeys.lastUsedTime | String   | Access key last used        | 
> | PrismaCloud.AccessKeys.status      | String   | Access key status           |
> | PrismaCloud.AccessKeys.role.id     | String   | User role id                | 
> | PrismaCloud.AccessKeys.role.name   | String   | User role name              |
> | PrismaCloud.AccessKeys.roleType    | String   | User role permission type   | 
> | PrismaCloud.AccessKeys.username    | String   | Access key user name        |

Reviewer 2: ```suggestion
| PrismaCloud.AccessKeys.id          | String   | Access key ID.               | 
| PrismaCloud.AccessKeys.name        | String   | The name of the access key.                        |
| PrismaCloud.AccessKeys.createdBy   | String   | The user who created the access key.                  | 
| PrismaCloud.AccessKeys.createdTs   | String   | Time access key was created. |
| PrismaCloud.AccessKeys.expiresOn   | String   | Access key expires on.       |
| PrismaCloud.AccessKeys.lastUsedTime | String   | The last time the access key was used.        | 
| PrismaCloud.AccessKeys.status      | String   | Access key status.           |
| PrismaCloud.AccessKeys.role.id     | String   | User role ID.       | 
| PrismaCloud.AccessKeys.role.name   | String   | User role name.              |
| PrismaCloud.AccessKeys.roleType    | String   | User role permission type.   | 
| PrismaCloud.AccessKeys.username    | String   | Access key user name.        |
```

                "name": "string",
                "createdBy": "string",
                "createdTs": "number",
                "lastUsedTime": "number",
                "status": "string",
                "expiresOn": "number",
                "role": {
                    "id": "string",
                    "name": "string"
                },
                "roleType": "string",
                "username": "string"
            },
            {
                "id": "string",
                "name": "string",
                "createdBy": "string",
                "createdTs": "number",
                "lastUsedTime": "number",
                "status": "string",
                "expiresOn": "number",
                "role": {
                    "id": "string",
                    "name": "string"
                },
                "roleType": "string",
                "username": "string"
            }
        ]
    }
}
```

`prisma-cloud-access-key-disable`

#### Input

| **Argument Name** | **Description**                                              | **Required** |
|-------------------|--------------------------------------------------------------| --- |

> | access-key              | Access key id                                                | Required | 

Reviewer 2: ```suggestion
| access-key              | Access key ID.                                                | Required | 
```

#### Command example
```!prisma-cloud-access-key-disable access-key=id```

`prisma-cloud-access-key-enable`

#### Input

| **Argument Name** | **Description**                                              | **Required** |
|-------------------|--------------------------------------------------------------| --- |

> | access-key              | Access key id                                                | Required | 

Reviewer 2: ```suggestion
| access-key              | Access key ID.                                                | Required | 
```

#### Command example
```!prisma-cloud-access-key-enable access-key=id```

`prisma-cloud-access-key-delete`

#### Input

| **Argument Name** | **Description**                                              | **Required** |
|-------------------|--------------------------------------------------------------| --- |

> | access-key              | Access key id                                                | Required | 

Reviewer 2: ```suggestion
| access-key              | Access key ID.                                                | Required | 
```

>### Next Page Token:
>token2

### Access Keys

***

> Access Keys are a secure way to enable programmatic access to the Prisma Cloud API. By default, only the System Admin has API access and can enable API access for other administrators. If you have API access, you can create up to two access keys.

Reviewer 2: ```suggestion
Access keys are a secure way to enable programmatic access to the Prisma Cloud API. By default, only the System Admin has API access and can enable API access for other administrators. If you have API access, you can create up to two access keys.
```

#### Command example
```!prisma-cloud-access-key-create name=MyNewKey```
#### Context Example
```json
{
    "PrismaCloud": {
        "AccessKeys": [
            {
                "Id": "Id",
                "Secret Key": "Secret Key"
            }
        ]
    }
}
```

`prisma-cloud-access-keys-list`

#### Input

| **Argument Name** | **Description** | **Required** |
|-------------------| --- | --- |
| access-key             | Returns the metadata of the access key that has the specified ID. | Optional | 
| limit           | Maximum number of entries to return. | Optional | 

#### Context Output

| **Path**                           | **Type** | **Description**             |
|------------------------------------|----------|-----------------------------|
| PrismaCloud.AccessKeys.id          | String   | Access key ID.               | 
| PrismaCloud.AccessKeys.name        | String   | The name of the access key.                        |
| PrismaCloud.AccessKeys.createdBy   | String   | The user who created the access key.                  | 
| PrismaCloud.AccessKeys.createdTs   | String   | Time access key was created. |

> | PrismaCloud.AccessKeys.expiresOn   | String   | Access key expires on.       |

Reviewer 2: ```suggestion
| PrismaCloud.AccessKeys.expiresOn   | String   | The time the access key expires.       |
```
